docs/design/auth.md

```diff
@@ -61,3 +61,9 @@ NOTE: The point is to make the token not *readily* available to developer contri
 The token is meant for tasks and scripts that are trusted by the build admin by (1) installing the task or (2) directing a build definition to run that script.  
 The goal is to avoid having the token accidentally leak in scripts.  
 Even then, the token will expire at the end of the job which helps mitigate any accidental exposure.
+
+## Keeping RSA private key - security implications for 'run once' agent setup
```

Comment:
does this really only apply for runonce?

Reply:
actually, another important use case to call out specifically is when the agent is run as a service -- do all the same rules apply there as well?

Reply:
Yep, they are actual for all scenarios - described this also, updated title